diff --git a/kernel/feature/selinux_hide.c b/kernel/feature/selinux_hide.c
index 1b34903..390a914 100644
--- a/kernel/feature/selinux_hide.c
+++ b/kernel/feature/selinux_hide.c
@@ -36,13 +36,13 @@ int security_sid_to_context_with_policy(struct selinux_policy *policy, u32 sid,
                                                u32 *scontext_len);
 void security_compute_av_user_with_policy(struct selinux_policy *policy, u32 ssid, u32 tsid, u16 tclass,
                                                  struct av_decision *avd);
-extern void security_dump_masked_av_fn(struct policydb *policydb,
+extern __weak void security_dump_masked_av_fn(struct policydb *policydb,
 				    struct context *scontext,
 				    struct context *tcontext,
 				    u16 tclass,
 				    u32 permissions,
 				    const char *reason);
-extern void context_struct_compute_av_fn(struct policydb *policydb,
+extern __weak void context_struct_compute_av_fn(struct policydb *policydb,
 				      struct context *scontext,
 				      struct context *tcontext,
 				      u16 tclass,